app/src/main/java/com/nimbl3/ui/main/MainViewModel.kt

```diff
@@ -10,11 +11,10 @@ import com.nimbl3.ui.main.data.Data
 import io.reactivex.Observable
 import io.reactivex.subjects.BehaviorSubject
 import io.reactivex.subjects.PublishSubject
-import javax.inject.Inject
 
 class MainViewModel
-@Inject constructor(private val repository: ApiRepository,
-                    private val schedulers: SchedulersProvider) : BaseViewModel(), Inputs, Outputs {
+@ViewModelInject constructor(private val repository: ApiRepository,
+                                 private val schedulers: SchedulersProvider) : BaseViewModel(), Inputs, Outputs {
```

Comment:
a bit weird format maybe you would need to recheck your IDE code format style? We should be using Kotlin default formatter

Reply:
yep, let me take a look on my IDE, it is freshly installed 😅